apps/bfd-server/bfd-server-war/src/main/config/server-start.sh

```diff
@@ -121,7 +121,7 @@ cd "${targetDirectory}/.."
 
 # Define all of the derived paths we'll need.
 workDirectory="${targetDirectory}/server-work"
-serverLauncher="${workDirectory}/$(ls ${workDirectory} | grep '^bfd-server-launcher-.*\.jar$')"
+serverLauncher="${workDirectory}/$(ls ${workDirectory} | grep '^bfd-server-launcher-' | grep -v '^bfd-server-launcher-.*\.zip$')/bfd-server-launcher.sh"
 serverPortsFile="${workDirectory}/server-ports.properties"
 serverLog="${workDirectory}/server-console.log"
 warArtifact="${targetDirectory}/$(ls ${targetDirectory} | grep '^bfd-server-war-.*\.war$')"
```

Comment:
General question - if we are using jetty then why not make our app a proper app with its own main and an embedded jetty?  That eliminates yet an other process an all the war file and other stuff related to running in a container.  Maybe that wouldn't work well for BFD but I've found it greatly simplifies things on other projects.  **Especially** when running in a docker container since it gives you a proper single process in container setup.

Reply:
And it makes debugging vastly easier as well since the IDE can just directly launch the app and debug it.

Reply:
With this PR we at least accomplish part of what you mentioned here if I understand it correctly -- we will only have one process running instead of the two that capsule requires which makes it a little bit easier to debug and launch from your IDE. The other part of what I believe you are suggesting is to also drop the war file and just register all of the servlets and filters inside of the main method? I'm guessing that this war file deployment was held over from the old jboss deployment. I'm interested in the idea of this simplification. @karlmdavis any thoughts or history to share?